src/utils/tezos/helpers.ts

```diff
@@ -186,11 +194,18 @@ export const operationsToBatchParams = ({
   type: operationsType,
   operations: originalOperations,
   sender,
+  ...restParams
```

Comment:
I am not sure what's going on here, but it's definitely wrong. 

<img width="453" alt="Screenshot 2024-05-28 at 23 21 15" src="https://github.com/trilitech/umami-v2/assets/129749432/fdf505cb-06b7-446a-aa27-99dc03eac3c6">